Production/python/scenarios.py

```diff
@@ -60,6 +60,13 @@ def __init__(self,sname):
                           jecfile="data/jec/Summer16_23Sep2016AllV3_DATA",
                           residual=True,
                           era="Run2_25ns")
+        elif sname == "2017PromptCollisions":

```

Comment:
* call this 2017BC (GT, etc. may change for later run periods)
* describe this scenario in README.md
* add an example to unitTest.py